src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -624,6 +625,16 @@ def check_sample_packing_w_xformers(cls, data):
 
         return data
 
+    @model_validator(mode="before")
+    @classmethod
+    def check_sample_packing_wo_flash(cls, data):
+        if data.get("sample_packing") and not data.get("flash_attention"):
```

Comment:
```suggestion
        if data.get("sample_packing") and not data.get("flash_attention") and not data.get("sup_attention"):
```
we also support sample packing for sdpa.

Reply:
fixed. thanks